app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Renderer/Abstract.php

```diff
@@ -77,6 +84,10 @@ public function renderExport(Varien_Object $row)
         return $this->render($row);
     }
 
+    /**
+     * @param Varien_Object $row
+     * @return string|null
```

Comment:
I'm not sure about return type of method `_getValue`. If getter is callable, is returned result of called method `return call_user_func($getter, $row)`.

Reply:
I have in fact made use of the `getter` param, yes, it can be either a method or a callback, but the return of both only make sense if the return type (null, string, or number) can be rendered in the cell of the grid; ie, the type should not be an object or array, for example. Therefore, string is the expected return of this function.